lampod-cli/src/args.rs

```diff
@@ -159,6 +160,15 @@ pub fn parse_args() -> Result<LampoCliArgs, lexopt::Error> {
         }
     }
 
+    if restore_wallet_flag {
+        println!("Enter the list of words for wallet restoration, separated by spaces:");
+        let mut var = String::new();
+        match std::io::stdin().read_line(&mut var) {
+            Ok(_) => mnemonic = Some(var.trim().to_string()),
+            Err(_) => todo!()
+        }
+    }
```

Comment:
This is not the place where to write this code, in this file there is the argument parsing, so I suggest you to write this code inside the main.rs.

In addition, we use `radicle-term` so we should see if they support some read input field from the docs https://docs.rs/radicle-term/latest/radicle_term/

e.g: https://docs.rs/radicle-term/latest/radicle_term/textarea/fn.textarea.html

Report the screenshot on discord to decide what will be the best for this usage